### ISSUE TITLE
Not compatible with Luajit/Lua 5.1

### ISSUE THREAD

**Issue by svipal** [stetre/moonvulkan#4]
cmd.c and debug_report_callback make use of the return value of lua_rawgeti, which returns void in above implementations

**Comment by stetre** [comment]
Hi. As stated in the README, it requires Lua >= 5.3. This implies that Lua 5.1 is not supported, and thus neither Luajit.

If that's the only compatibility issue, however, maybe it could be fixed by just adding the lua-compat module.

EDIT: I added lua-compat and while I was at it, I also added it to [MoonGLFW](https://github.com/stetre/moonglfw). I tried it with Lua 5.1 on Ubuntu and it compiles and loads just fine, but many examples won't work because they use 5.3 features for bitwise operations.
If you use it, keep in mind that the 'unsupported' for Lua < 5.3 still holds, though.
